File: src/Cxx/PolyData/Stripper.cxx
#include <vtkSmartPointer.h>
#include <vtkPolyData.h>
#include <vtkSphereSource.h>
#include <vtkStripper.h>

int main(int, char *[])
{
  vtkSmartPointer<vtkSphereSource> sphereSource = 
    vtkSmartPointer<vtkSphereSource>::New();
  sphereSource->Update();
  
  std::cout << "Number of cells before stripping: " << sphereSource->GetOutput()->GetNumberOfCells() << std::endl;
  
  vtkSmartPointer<vtkStripper> stripper = 
      vtkSmartPointer<vtkStripper>::New();
  stripper->SetInputConnection(sphereSource->GetOutputPort());
  stripper->Update();
  
  std::cout << "Number of cells after stripping: " << stripper->GetOutput()->GetNumberOfCells() << std::endl;
  
  return EXIT_SUCCESS;
}

File: src/Java/PolyData/Stripper.java
import vtk.vtkSphereSource;
import vtk.vtkStripper;
import vtk.vtkNativeLibrary;

class Stripper{  
	
  static {
    if (!vtkNativeLibrary.LoadAllNativeLibraries()) {
      for (vtkNativeLibrary lib : vtkNativeLibrary.values()) {
        if (!lib.IsLoaded()) {
          System.out.println(lib.GetLibraryName() + " not loaded");
        }
      }
    }
    vtkNativeLibrary.DisableOutputWindow(null);
  }
	
	
  public static void main(String args[]){  
    	
    vtkSphereSource SphereSource = new vtkSphereSource();
    SphereSource.Update();
		
    System.out.println("Number of cells before stripping:" + SphereSource.GetOutput().GetNumberOfCells() );
	
			  
    vtkStripper Stripper = new vtkStripper();
    Stripper.SetInputConnection(SphereSource.GetOutputPort());
    Stripper.Update();

			  
    System.out.println("Number of cells after stripping:" + Stripper.GetOutput().GetNumberOfCells() );
			  
		  		  
  }  
}   
